### DISCUSSION TITLE
Account Suspension.

### DISCUSSION THREAD

**Discussion by ragilsuanda** [community/community#144839] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Visual Studio

### Body

 Account Suspension.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ragilsuanda, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to GitHub account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). For Visual Studio, you will need to go through their support options. 

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
